stacks-signer/src/runloop.rs

```diff
@@ -266,7 +266,11 @@ impl RunLoop {
                     .insert(reward_index, Signer::from(new_signer_config));
                 debug!("Reward cycle #{reward_cycle} Signer #{signer_id} initialized.");
             } else {
-                warn!("Signer is not registered for reward cycle {reward_cycle}. Waiting for confirmed registration...");
+                if current {
```

Comment:
So is `current` only controlling the contents of the warning message here?  If so, then can we omit `current` as an argument, and instead compute whether or not `reward_cycle` is the last reward cycle based on our last-seen burnchain block?

Reply:
We are not currently tracking our last-seen burnchain block, so that would require a call to our stacks-node (which the caller of this function has already done).

Reply:
Okay, can you add a TODO item to remove this once the signer starts tracking the last-seen burnchain block?  I think that's something you're going to want to be doing anyway?

Reply:
Added TODO in 6e4d9cce7